entity/fitsQAUberCreateEntity.java

```diff
@@ -0,0 +1,126 @@
+package samplelearning;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+import com.github.javafaker.Faker;
+
+
+public class fitsQAUberCreateEntity {
```

Comment:
Please add documentation. What is this code doing?  Place the JIRA id to the test case.

Reply:
This code(fitsQAUberCreateEntity) creates entities as SA.
Below the details how it works as part of  documentation.
Pre-requisite:- chrome web driver
Steps:-
Log-In as SA
Navigate Entities.
Add Entity.
Create Entity.
Subscription adjust.
Pay pal sand box Login and payment.